budget_allocation/models/base_budget_move.py

```diff
@@ -88,6 +88,14 @@ def check_budget_constraint(self, docline):
                     ba_line_group = self._get_ba_line_group(
                         budget_allocation_lines, obj_group
                     )
+                    # Spent budget is not allocated
+                    if not ba_line_group:
+                        raise UserError(
+                            _(
+                                "Can not spent amount becase budget is not "
```

Comment:
```suggestion
                                "Can not spend amount because the budget is not "
```

Reply:
okay, I will fix it on pr https://github.com/ecosoft-odoo/budgeting/pull/176